### ISSUE TITLE
Importing and Exporting Diagrams

### ISSUE THREAD

**Issue by josephcarr** [open-source-labs/Svelvet#505]
I saw on Svelvet 5.0 there was added functionality to import and export diagrams as json objects. Is there still functionality for that because I have not found documentation for it since 6.0.

**Comment by magick93** [comment]
Yes, I'd be interested docs on the recommended approach for persisting, importing and exporting of flows.

**Comment by madhudson** [comment]
This would be extremely useful. They had something like this in earlier versions

**Comment by Br3nnabee** [comment]
It's something that's apparently being worked on, with the node positions already able to be read; see #108 and #365.
